packages/tests-postman/index.js

```diff
@@ -171,31 +170,43 @@ const urlExist = async (url) => {
 // run a base sanity check...
 async function livenessCheck() {
   console.log('Quick liveness tests on service providers...');
-  for (const serv of serviceData) {
-    if (program.opts().names.includes(serv.name) || program.opts().names.includes('all')) {
-      console.log('Checking', serv.name);
-      const spExists = await urlExist(serv.serviceProvider.provider.url); // eslint-disable-line no-await-in-loop
-      const didExists = await urlExist(`https://${serv.serviceProvider.baseURL}/.well-known/did-configuration.json`); // eslint-disable-line
-      console.log('\t', serv.serviceProvider.provider.url, spExists);
+
+  const { names } = program.opts();
+  const services = serviceData.filter((serv) => names.includes('all') || names.includes(serv.name));
+
+  const results = [];
+
+  for (const serv of services) {
+    console.log('Checking', serv.name);
+
+    const urlSVC = serv.serviceProvider.provider.url;
+    const urlDID = `https://${serv.serviceProvider.baseURL}/.well-known/did-configuration.json`;
```

Comment:
I dream of a change to this in the future.

Reply:
This. 100%.

Reply:
see #55